src/index.js

```diff
@@ -0,0 +1,273 @@
+        );
+      return;
+    }
+
+    for (let i = 0; i < arrayExpressionPath.node.elements.length; i++) {
+      const element = arrayExpressionPath.get("elements", i)[i];
+      element.replaceWith(computeClassName(element, cssmodule));
+    }
+  };
+
+  const computeClassName = (value, cssmodule) => {
+    if (t.isStringLiteral(value)) {
+      const values = value.node.value.split(" ");
+      return values.length === 1
+        ? t.memberExpression(cssmodule, t.stringLiteral(values[0]), true)
+        : t.templateLiteral(
+          spacedTemplateElements(values.length + 1),
+          values.map((v) => t.memberExpression(cssmodule, t.stringLiteral(v), true))
+      );
+    } else if (t.isIdentifier(value)) {
+      // TODO: need to validate what type of node this identifier refers to
+      return t.memberExpression(cssmodule, value.node, true);
+    }
+  };
+
+  /**
+   * Updates a callExpression value with the most appropriate CSS Module lookup.
+   *
+   * @param {Path} callExpression <jsx className={value()} />
+   * @param {Node<Identifier>} cssmodule cssmodule identifier
+   */
+  const replaceCallExpression = (callExpression, cssmodule) => {
+    if (isArrayJoin(callExpression)) {
+      return replaceArrayJoinElements(callExpression, cssmodule);
+    }
+
+    // this is just mean
+    callExpression.replaceWith(
+      t.callExpression(
+        t.memberExpression(
+          t.callExpression(
+            t.memberExpression(
+              t.callExpression(
+                t.memberExpression(callExpression.node, t.identifier("split"), false),
+                [t.stringLiteral(" ")]
+              ),
+              t.identifier("map")
+            ),
+            [t.arrowFunctionExpression(
+              [t.identifier("i")],
+              t.memberExpression(cssmodule, t.identifier("i"), true)
+            )]
+          ),
+          t.identifier("join"),
+          false
+        ),
+        [t.stringLiteral(" ")]
+      )
+    );
+  };
+
+  const updateClassName = (value, cssmodule) => {
+    if (t.isStringLiteral(value)) {
+      return value.replaceWith(computeClassName(value, cssmodule));
+    } else if (t.isIdentifier(value)) {
+      return value.replaceWith(computeClassName(value, cssmodule));
+    } else if (t.isCallExpression(value)) {
+      return replaceCallExpression(value, cssmodule);
+    } else {
+      console.log("TODO: updateClassName for %s", value.type);
+    }
+  };
+
+  /**
+   * Updates a JSX className value with the most appropriate CSS Module lookup.
+   *
+   * @param {Path} value <jsx className={value} />
+   * @param {Node<Identifier>} cssmodule cssmodule identifier
+   */
+  const updateJSXClassName = (value, cssmodule) => {
+    if (!t.isJSXExpressionContainer(value)) {
+      value.replaceWith({
+        type: "JSXExpressionContainer",
+        expression: value.node
+      });
+    }
+    updateClassName(value.get("expression"), cssmodule);
+  };
+
+  const buildRequire = template(`
```

Comment:
For a second I got excited that you had found a way to pass a variable into a template string. Nope, no dice.